source/_components/notify.group.markdown

```diff
@@ -15,12 +15,12 @@ ha_qa_scale: internal
 
 The `group` notification platform allows you to combine multiple `notify` platforms into a single service.
 
-To use this notification platform in your installation, add the following to your `configuration.yaml` file:
```

Comment:
Not sure why this became a title. This is kinda not matching our other documentation.